Src/Plugins/Tests.Security/EndToEndTests.cs

```diff
@@ -104,16 +97,24 @@ protected EndToEndTests(ITestOutputHelper outputHelper) : base(outputHelper)
         private Dictionary<string, string> MultiThreadedConstructTestOutputs(IEnumerable<string> inputResourceNames, List<string> inputFiles)
         {
             var results = new Dictionary<string, string>();
+
             var dict = new Dictionary<string, Task<string>>();
```

Comment:
>dict [](http://example.com/codeflow?start=16&length=4)

I know this wasn't added by this PR, just pointing out that this fails to adhere to naming conventions.